modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -115,21 +121,23 @@ If an operation has multiple message schemas associated in the specification, th
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | Name of the configuration to use | None | Yes 
-| Content | Binary | None | #[payload] | No  
-| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | Yes 
+| Configuration | String | Name of the configuration to use | None | Yes
+| Content | Binary | The binary content to be processed | #[payload] | No
+| Disable Content Validation | Boolean | Indicates if validation of the content is disabled | `False` | No
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | Yes
 | Channel Name | String | Channel name | None | Yes 
 | Server Name | String | Server name | None | Yes 
-| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
-| Target Value | String | Expression that is evaluated against the operation's output and the result of that expression is stored in the target variable | #[payload] | No
+| Target Variable | String | Name of a variable on which the operation output is placed | None | No 
```

Comment:
is "on" correct here?

Reply:
The operation output is placed ON the name of the variable. Isn't "on" the appropriate preposition? Thanks!